alacritty_terminal/src/term/mod.rs

```diff
@@ -1400,7 +1398,7 @@ impl<T> Term<T> {
     }
 
     /// Get rendering information about the active cursor.
-    fn renderable_cursor<C>(&self, config: &Config<C>) -> RenderableCursor {
+    fn renderable_cursor<C>(&self, config: &Config<C>, show_cursor: bool) -> RenderableCursor {
```

Comment:
Hiding it here rather than the iterator actually makes a lot more sense anyways.